### DISCUSSION TITLE
使用腾讯云COS无法上传图片

### DISCUSSION THREAD

**Discussion by maajiko** [lsky-org/lsky-pro#466] [问答] [answered]
腾讯云COS无法上传图片反馈，上传失败服务异常，默认本地策略，上传没有问题

查看AppId，SecretId，SecretKey，多没有问题

应该怎么解决![image](https://user-images.githubusercontent.com/83452696/177078254-cdd8845d-8a79-43e5-a808-c9ef082c878d.jpeg)

[图床](https://img.maajiko.cn/)

**Comment by 0xxb** [comment, marked as answer]
大概率是这个原因：https://github.com/lsky-org/lsky-pro/issues/365#issuecomment-1084097435

**Comment by maajiko** [comment]
> 大概率是这个原因：https://github.com/lsky-org/lsky-pro/issues/365#issuecomment-1084097435

这又是什么问题？🤣第1次安装使用

储存桶，已开通套餐，非子账号
依旧提示上传失败![image](https://user-images.githubusercontent.com/83452696/177086221-5a578c4f-335c-4dbf-84ca-e581b814d4e6.png)
